apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -112,6 +113,10 @@ const VERIFIED_MOBILE_NUMBERS_ATTRIBUTE: string =
     ProfileConstants.SCIM2_SCHEMA_DICTIONARY.get("VERIFIED_MOBILE_NUMBERS");
 const VERIFIED_EMAIL_ADDRESSES_ATTRIBUTE: string =
     ProfileConstants.SCIM2_SCHEMA_DICTIONARY.get("VERIFIED_EMAIL_ADDRESSES");
+const EMAIL_VERIFIED_ATTRIBUTE: string =
```

Comment:
This attribute is used to check the primary email is verified right? If yes, can we rename like below for better readability. WDYT?

```suggestion
const PRIMARY_EMAIL_VERIFIED_ATTRIBUTE: string =
```

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/8137/commits/e8c612c9d1e6faea387ac942e84ad59d061f85f2